mhr_api/src/mhr_api/resources/v1/other_registrations.py

```diff
@@ -84,7 +84,7 @@ def post_other_registration(mhr_number: str):
             return resource_utils.unauthorized_error_response(account_id)
         # Try to fetch summary registration by mhr number
         registration = MhrRegistration.find_summary_by_mhr_number(account_id, mhr_number)
-        if registration is None:
+        if not registration:
             return resource_utils.not_found_error_response('Manufactured Home registration', mhr_number)
 
         # Check if duplicate.
```

Comment:
Is this related to the PR? 
Was this branch potentially cut from your previous API pr branch?

Reply:
This was from another PR, got pulled in when I rebased BC it hasn't been merged yet

Reply:
I think